net.solarnetwork.node.weather.ibm.wc/META-INF/MANIFEST.MF

```diff
@@ -19,6 +19,7 @@ Import-Package: com.fasterxml.jackson.core;version="[2.4,3.0)",
  org.eclipse.gemini.blueprint.compendium.cm;version="[2.0,3.0)",
  org.joda.time;version="[2.4,3.0)",
  org.joda.time.base;version="[2.4,3.0)",
+ org.joda.time.format;version="2.9.7",
```

Comment:
Should use the same version range as the other Joda Time imports, e.g. `[2.4,3.0)`.

Reply:
Oops, went the wrong way with this change. All 3 `org.joda.*` imports should use a version range like `[2.4,3.0)`, *not* `2.9.7`.